src/core/forces.hpp

```diff
@@ -60,10 +61,11 @@ void init_forces_ghosts();
  *  <li> Calculate long range interaction forces
  *  </ol>
  */
-void force_calc();
+void force_calc(const ParticleRange &particles);
```

Comment:
you mean, we should also extract the ```cell_structure``` global and the ```ghost_cells```?

Reply:
You can either leave the globals, and refactor this latter, or put the `{local, global}_cells`, potentially as dummy methods that return the globals into `CellStructure` and pass that.
They should not passed next to each other, because you actually have to have the cells that belong to the cell structure that is passed, and the particles that belong to those cells.